src/apps/appstore/views/marketClusterLink.html

```diff
@@ -1,3 +1,6 @@
+<div class="market-beta-notice">
+	<i class="fa fa-info-circle"></i> {{ i18n.marketplace.betaNotice }}
+</div>
```

Comment:
I think it would be better to use the [monsterPanelText](https://docs.2600hz.com/ui/docs/handlebars/#monsterpaneltext) handlebars helper that is available in Monster UI, which allows to display informational messages using the UI look and feel.

Reply:
@guillegr123 If I'm trying to use `monsterPanelText`, and it is adding space here:
<img width="1079" alt="image" src="https://user-images.githubusercontent.com/4894215/180583937-343e13d1-d227-4732-a2ad-520595c63e37.png">

If I remove `clearfix` from `monster-panel-text-header` element it would fix it:

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/4894215/180583993-a6355cd2-7a03-4723-bc9e-5540bd8efcb1.png">

Reply:
okay I fixed it with by setting `overflow:hidden` and applying some padding so its border will show.

Reply:
Sorry @icehess I did not see your messages until today :( Looks good after your fixes 👍